bdk_core/tests/test_keychain_tracker.rs

```diff
@@ -22,9 +22,11 @@ fn no_checkpoint_and_then_confirm() {
 
     assert_eq!(chain.apply_checkpoint(checkpoint.clone()), ApplyResult::Ok);
 
+    // TODO: Default SparseChain will create one checkpoint with all 0 values.
+    // Convert the default into genesis block of each network.
```

Comment:
I don't think this is a good TODO. What do we gain by doing this?

Reply:
I removed the TODO but would like to know in what direction you were thinking?

Reply:
I was mostly thinking of if its makes sense to start the sparse chain with 0 value blockhash.. Ideally i felt it should point to the genesis block.. But also as the genesis block will not contain any transactions and we are identifying checkpoints with cumulative transaction hashes it also doesn't make much difference.. So in that light it does make sense to remove the todo..